src/org/usfirst/frc4904/robot/subsystems/CubeIO.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class CubeIO extends Subsystem {
+
+	public static Motor roller;
```

Comment:
Motors shouldn't be stored as a static variable unless they are inside RobotMap. This is not a general programming practice, and yes usually the static variables are stored in the most relevant class. However for our purposes it makes more sense to store them in one file to make emergency edits at competition, on the field, etc.